### PR TITLE
Initial ecto subscribe v3

### PR DESCRIPTION
**Pull request by 0xAX**

### CODE REVIEW

**Comment by liveforeverx** [comment]
Please, rebase your commits, that every commit have meaningful message

File: test/db/dbhelper.ex [hunk 1]
```diff
@@ -0,0 +1,35 @@
+defmodule Ecto.Subscribe.Test.DbHelper do
```
Comment (liveforeverx) [inline test/db/dbhelper.ex:1]: The whole file and on some other places should be replaced with EctoIt

File: test/ecto_subscribe_test.exs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  use Ecto.Repo,
+  otp_app: :ecto_subscribe,
+  # TODO remove it
+  adapter: TestDbHelper.get_adapter
+end
+
+defmodule EctoSubscribeTest do
+  use ExUnit.Case
+
+  {adapter, url} = case Mix.env do
```
Comment (liveforeverx) [inline test/ecto_subscribe_test.exs:35]: replace with EctoIt start

**Comment by 0xAX** [comment]
will resend

**Pull request closed without merging**